src/main/webapp/app/course/manage/course-group.component.ts

```diff
@@ -272,4 +277,35 @@ export class CourseGroupComponent implements OnInit, OnDestroy {
         this.rowClass = className;
         setTimeout(() => (this.rowClass = undefined));
     };
+    /**
+     * Method for exporting the csv with the needed data
+     */
+    exportUsersInformation() {
+        if (this.allCourseGroupUsers.length > 0) {
```

Comment:
Could you please make sure at this point that the group users are only loaded in the client if the user is an instructor?

Reply:
Solved. This is implemented in the src/main/webapp/app/course/manage/course-group.component.html